pom.xml

```diff
@@ -8,7 +8,7 @@
 
   <groupId>com.cloudbees.jenkins.plugins</groupId>
   <artifactId>docker-custom-build-environment</artifactId>
-  <name>CloudBees Docker Custom Build Environment Plugin</name>
+  <name>Docker Custom Build Environment Plugin. Maintained by CloudBees, Inc.</name>
```

Comment:
@recena I think the decisions you refer to are outdated. Please follow-up with OSS Team  internally before going forward.